lib/actions/adapterActions.js

```diff
@@ -123,10 +123,11 @@ const secKeyset = {
 
 let throttledValueChangedDispatch;
 
-function adapterOpenedAction(adapter) {
+function adapterOpenedAction(adapter, port, versionInfo) {
```

Comment:
The `port` should be removed from the list of parameters here, and from the list of arguments when dispatching the action.